.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "Quagga"]
 	path = Quagga
-	url = https://github.com/Fibbing/Quagga.git
+	url = https://github.com/lferran/Quagga.git
```

Comment:
True. I wanted to select specific commits for the pull request, and obviously leave this one out.. but I didn't find how.